keyboards/treadstone48/keymaps/default/keymap.c

```diff
@@ -0,0 +1,303 @@
+#include QMK_KEYBOARD_H
+#include "keymap_jp.h"
+#include "bootloader.h"
+#ifdef PROTOCOL_LUFA
+#include "lufa.h"
+#include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+#include "../common/oled_helper.h"
```

Comment:
A lot of these includes may not actually be necessary.

And the ones that are.... they should probably be moved to the keyboard's header file, instead of having them here.

Reply:
Already moved include ssd1306.h from keymap.c to treadstone48.h
but not moved oled_helper.h because the content of OLE display is to be selectable by the user.